src/cargo/lib.rs

```diff
@@ -1,7 +1,7 @@
 #![cfg_attr(test, deny(warnings))]
 #![warn(rust_2018_idioms)]
-// while we're getting used to 2018
-// Clippy isn't enforced by CI, and know that @alexcrichton isn't a fan :)
+// While we're getting used to 2018:
+// Clippy isn't enforced by CI, and I know that @alexcrichton isn't a fan.
```

Comment:
I think the meaning here isn't "I know", it's "you should know".

Reply:
Okay. That was somewhat ambiguous. I've made it clearer now I think.